test/node/testCommonFunctions.js

```diff
@@ -111,4 +111,38 @@ describe('commonFunctions.js', function(){
       expect(finalBasal.duration).to.equal(0);
     });
   });
+
+  describe('computeMillisInCurrentDay', function(){
+
+    var basal;
+    beforeEach(function(){
+      basal = builder.makeScheduledBasal()
+        .with_deviceTime('2015-11-05T17:00:00')
+        .with_time('2015-11-05T17:00:00.000Z')
+        .with_rate(0.3)
+        .with_scheduleName('Test')
+        .with_conversionOffset(0)
+        .with_timezoneOffset(0);
+    });
+
+    it('returns milliseconds in current day', function(){
+      expect(common.computeMillisInCurrentDay(basal)).to.equal(61200000);
+    });
+
+    it('rounds to nearest 15 minutes for clock skew', function(){
+      basal.with_time('2015-11-05T17:05:00.000Z')
+           .with_conversionOffset(420000);
+      console.log(basal);
+      expect(common.computeMillisInCurrentDay(basal)).to.equal(61200000);
+    });
+
+    it('checks that rounding does not result in more than 24 hours', function(){
```

Comment:
Can you explain how this tests that it doesn't result in > 24 hours? How would you get this test to fail by changing the logic of `computeMillisInCurrentDay`? I'm not seeing it, but it admittedly does my head in a bit to try to think through this stuff with your positive offset from UTC since I'm used to thinking about negative offsets.

In any case, this is also not really testing this at the right point. I really intended for tests to be added to sundial...but now looking at the `sundial.getMsFromMidnight` I'm fairly convinced that it can't screw up. The trickier case where I had to be careful about this is when a milliseconds-from-midnight function is taking a timezone as input, not a timezone offset. (Because the timezone can be associated with more than one offset in a single day, so you have to be careful about how you're finding the previous midnight that you're subtracting...)

(The other thing that's getting in the way of readability here, IMO, is using the `conversionOffset` in more than one test set up... I think it'd be better to isolate what's being tested a bit more and if needed have one "kitchen sink" test that throws all the weird things together.)